README.md

```diff
@@ -55,17 +55,17 @@ See also:
 * [Throughs](./docs/throughs/index.md)
 * [Sinks](./docs/sinks/index.md)
 
-### Source (aka, Readable)
+### Source (readable stream that produces values)
 
-The readable stream is just a `function read(end, cb)`,
+A Source is a function `read(end, cb)`,
 that may be called many times,
-and will (asynchronously) `cb(null, data)` once for each call.
+and will (asynchronously) call `cb(null, data)` once for each call.
 
 To signify an end state, the stream eventually returns `cb(err)` or `cb(true)`.
-When indicating a terminal state, `data` *must* be ignored.
+When signifying an end state, `data` *must* be ignored.
 
 The `read` function *must not* be called until the previous call has called back.
-Unless, it is a call to abort the stream (`read(truthy, cb)`).
+Unless, it is a call to abort the stream due to an error (`read(truthy, cb)`).
```

Comment:
this edit changes the meaning. It can actually abort the stream in absence of an error.

Reply:
ah I see, so `read` can be called early if 
1) there is an error (`read(Error, cb)`)
2) no error but the sink is no longer interested in values (`read(true, cb)`)

Is that right?

Reply:
@dominictarr could you clarify this? I can then fix this and we can merge